### DISCUSSION TITLE
Can open-watcom for Linux load/use .so files?

### DISCUSSION THREAD

**Discussion by eddavis2** [open-watcom/open-watcom-v2#836] [Q&A]
Can open-watcom for Linux load/use .so files?

I've read that it cannot create them, but can it use existing ones?

If so, is there an example somewhere, e.g., wcl switches/options to use?

Thank you.

**Comment by jmalak** [comment]
Yes, OW cannot create shared libraries for Linux.
Linker could be capable to use shared library, but it is not tested anyway.
You must use appropriate wcl option to add linker command "library .....".
